### PR TITLE
Fixing broken links

### PR DESCRIPTION
**Pull request by xomiamoore**

### CODE REVIEW

**Comment by xomiamoore** [comment]
@camunda-community-hub/devrel if anyone has time for a quick link check :)

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ We’ve put together some commonly asked questions and answers about the Camunda
 ### Contributor resources & documentation
 * [Contributor guide](https://github.com/camunda-community-hub/community/blob/main/CONTRIBUTING.MD)
 * [Issue triage and labeling](https://github.com/camunda-community-hub/community/blob/main/issue-triage.md)
-* [Maintainer & reviewer expectations]([#maintainer-reviewer-expectations.md](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md))
+* [Maintainer & reviewer expectations](https://github.com/camunda-community-hub/community/blob/main/maintainer-reviewer-expectations.md)
```
Comment (christinaausley) [inline README.md:36]: Does `[Maintainer & reviewer expectations](./maintainer-reviewer-expectations.md)` work here?

Comment (xomiamoore) [reply README.md:36]: This one *should* work if I do that, but I was avoiding relative links to make sure I got it right. Is it better practice to use relative links?

Comment (christinaausley) [reply README.md:36]: If the files are within the same repo, yes.

Comment (xomiamoore) [reply README.md]: Updated!

**Review by christinaausley — APPROVED** [review]

**Pull request merged**